### DISCUSSION TITLE
Graph line style in qucs-s macOS

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#1151] [General]
In the graph setup, one can select various line styles such as solid line, dash line, dot line, long dash line.
In my 24.4.0 for MacOS, none of these selections makes any difference, all lines are still solid.
It would be nice if it could be made working.

**Comment by ra3xdh** [comment]
This should be fixed by #1042. Try to update to v24.4.1

**Comment by bnilsson11** [comment]
Indeed it is fixed.
Thanks.
